src/cmake/modules/FindOpenEXR.cmake

```diff
@@ -71,16 +71,29 @@ endif ()
 
 # List of likely places to find the libraries -- note priority override of
 # OPENEXR_CUSTOM_LIB_DIR and ${OPENEXR_HOME}/lib.
-set (GENERIC_LIBRARY_PATHS 
-    ${OPENEXR_CUSTOM_LIB_DIR}
-    ${OPENEXR_HOME}/lib
-    ${ILMBASE_HOME}/lib
+
+# If there's no OPENEXR_HOME or ILMBASE_HOME, then the path will point to
+# "/lib", which may not always be wanted/expected.
+if (${OPENEXR_CUSTOM_LIB_DIR})
```

Comment:
Good thing I tested at work, where we do set OPENEXR_CUSTOM_LIB_DIR!

This is the wrong syntax. It should be

    if (OPENEXR_CUSTOM_LIB_DIR)

And similar below.

I'll fix it on my end and merge.